lib/kafka_rest_client.js

```diff
@@ -263,15 +266,15 @@ KafkaRestClient.prototype.getTopicRequestBody = function getTopicRequestBody(pro
     }
     self.mainHttpClient.get(endpoint, function handleGetCall(err, res, body) {
         if (err) {
-            callback(err, null);
-        } else {
-            callback(null, body);
+            return callback(err, null);
         }
+        callback(null, body);
```

Comment:
`return callback(null, body)` to be consistent with what you do in other places?

Reply:
basically the lint rule sees a "callback" it asks for "return" in front of it, unless it is last line of function.
Old code is
`if (err) callback(err, null); else callback(null, body);`
new code is 
```
if (err) {return callback(err, null);};
 callback(null, body);
```
Should be the same logic.

Reply:
got it, curious why the lint rule prefers no return in the last line